cra-config.yaml

```diff
@@ -0,0 +1,7 @@
+version: "v1"
+CRA_TARGETS:
+  - CRA_TARGET: "solutions/basic"
```

Comment:
should we not be scanning `solutions/ibm-catalog/sap-ready-to-go` since this is the DA solution?

Reply:
That cannot be scanned, It requires a schematics prereq id.

Reply:
Is there a way something like: pre script: which installs catalog tile using ibmcloud catalog install powervs infra:
after this the schmeatics workspace id is passed for the PRs and CRA SCAN ?

and in post script after the test we can do ibmcloud schematics destroy --id 

This way cra scan and PR runs can be validated here

Reply:
we don't want to deploy any resources as part of CRA. CRA is only designed to run on a terraform plan. Does the plan actually need a valid schematics prereq id? Does it do a data lookup or something?

Reply:
Yes it does require to look up resources. Data sources are used

Reply:
ok, then not much we can do here. I'll merge as is